### PR TITLE
Add lot of missing locations

### PR DESCRIPTION
**Pull request by Saentist**
Added missing satellite/cable locations

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-dinky
```
Comment (Flole998) [inline _config.yml]: Why exactly is this necessary?

Comment (Saentist) [reply _config.yml]: Save to remove this line it this is github template.

Compare local git to 
https://git.linuxtv.org/dtv-scan-tables.git/

**Comment by Flole998** [comment]
Impossible to merge due to conflicts

**Comment by Saentist** [comment]
Make demonstration how can be merged with
git://linuxtv.org/dtv-scan-tables.git
https://git.linuxtv.org/dtv-scan-tables.git

there is no conflicts there is so many changes in time, that git thinks is conflict

**Comment by Flole998** [comment]
> there is no conflicts there is so many changes in time, that git thinks is conflict

Of course there are conflicts, git doesn't show false positive conflicts..... Come up with a proper PR that properly merges those 2 branches by basically manually cherry-picking or rejecting each commit. Everything else is incorrect.

**Pull request closed without merging**